stl/inc/yvals_core.h

```diff
@@ -1332,9 +1332,14 @@
 
 #define __cpp_lib_remove_cvref            201711L
 #define __cpp_lib_semaphore               201907L
-#define __cpp_lib_shift                   201806L

```

Comment:
In this file you should also add a comment about implementing the feature.

See https://github.com/microsoft/STL/blob/7b2c54be012f998736ebb6b72c7804a3f6c5f8e5/stl/inc/yvals_core.h#L282-L303